app/components/primer/dropdown/menu_component.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Primer
+  module Dropdown
```

Comment:
Can we define this inside the DropdownComponent? Like:

```suggestion
  class DropdownComponent
```

Reply:
I wanted to avoid being so verbose `Primer::DropdownComponent::MenuComponent`, the `Component` repetition bothers me

Reply:
Hmm. I guess what concerns me is that we now have a new module here. While it _is_ verbose, the nesting of the component naming does reflect the relationship. This is definitely a case where I wish we didn't have `Component` in the name of our components...

Reply:
I think that having `Dropdown` (the parent component name) is enough to show the relationship. I really want to avoid being too verbose